pkg/skaffold/deploy/helm.go

```diff
@@ -29,16 +29,14 @@ import (
 )
 
 type HelmDeployer struct {
-	*config.DeployConfig
-	kubeContext string
+	*KubectlDeployer
```

Comment:
Why does HelmDeployer embed KubectlDeployer?

Reply:
I thought it made sense but I'll revert it